src/views/Migrate.vue

```diff
@@ -110,17 +275,312 @@ export default class Migrate extends Vue {
+        padding: 2.5rem;
+        border-radius: .5rem;
+        font-size: 2.375rem;
+        font-weight: 600;
+        text-align: center;
+        position: relative;
+    }
+
+    .check-box {
+        transition: opacity .5s .5s ease-in;
+    }
+
+    .nq-button.activate {
+        transition:
+            transform 450ms cubic-bezier(.25,0,0,1),
+            box-shadow 450ms cubic-bezier(.25,0,0,1),
+            opacity 450ms cubic-bezier(.25,0,0,1);
+    }
+
+    .check-box input {
+        position: absolute;
+        left: -9999rem;
+        opacity: 0;
+    }
+
+    .check-box label {
+        display: inline-flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        padding: 1.5rem 2.5rem;
+        cursor: pointer;
+        border-radius: 4rem;
+    }
+
+    .check-box .checkcircle {
+        border: solid .25rem rgba(255, 255, 255, 0.6);
+        border-radius: 50%;
+        width: 3.5rem;
+        height: 3.5rem;
+        margin-right: 2rem;
+        position: relative;
+        flex-shrink: 0;
+    }
+
+    .check-box .checkcircle .nq-icon {
+        display: none;
+    }
+
+    .check-box .checkcircle::after {
+        content: "";
+        position: absolute;
+        left: -0.875rem;
+        top: -0.875rem;
+        right: -0.875rem;
+        bottom: -0.875rem;
+        border: 0.25rem solid rgba(255, 255, 255, 0.6);
+        border-radius: 50%;
+        opacity: 0;
+    }
+
+    .check-box input:focus ~ .checkcircle::after,
+    .check-box input:active ~ .checkcircle::after {
+        opacity: 1;
+    }
+
+    .check-box input:checked ~ .checkcircle {
+        background: rgba(255, 255, 255, 0.6);
+        border: none;
+    }
+
+    .check-box input:checked ~ .checkcircle::after {
+        left: -0.625rem;
+        top: -0.625rem;
+        right: -0.625rem;
+        bottom: -0.625rem;
+    }
+
+    .check-box input:checked ~ .checkcircle .nq-icon {
+        position: absolute;
+        display: block;
+        font-size: 3rem;
+        margin-left: -1.125rem;
+        left: 50%;
+        bottom: .75rem;
+    }
+
+    .transition-fade-enter,
+    .transition-fade-leave-to {
+        opacity: 0;
+    }
+
+    /*
+    Because Vue does virtual-DOM diffing, pages are not replaced as complete components but rather
+    parts of the pages are replaced, e.g. the footer content (while the footer element itself
+    stays the same).
+
+    This has a negative visual effect on the check-box, because it has a transition applied
+    which fades it out when it is de-rendered. Because of the DOM-diffing instead of full element
+    replacing, the check-box is rendered fading-out next to the intro page's "Prepare for update"
+    button when clicking on the back arrow.
+
+    These styles here prevent that from happening by force-hiding them.
+
+    If you know a better method to force Vue to throw out an element instead of fading it out,
+    please make yourself known.
```

Comment:
By adding a `key` to the vue elements, this shouldn't happen.

Reply:
I tried `:key`... :man_facepalming: